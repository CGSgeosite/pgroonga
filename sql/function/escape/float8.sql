SELECT pgroonga.escape(29.2929::float8);
SELECT pgroonga.escape(-29.2929::float8);
