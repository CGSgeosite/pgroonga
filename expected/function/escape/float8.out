SELECT pgroonga.escape(29.2929::float8);
 escape  
---------
 29.2929
(1 row)

SELECT pgroonga.escape(-29.2929::float8);
  escape  
----------
 -29.2929
(1 row)

